internal/plugins/manifests/v2/init.go

```diff
@@ -310,4 +320,17 @@ catalog-build: opm ## Build a catalog image.
 catalog-push: ## Push a catalog image.
 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
 `
+
+	// TODO: remove it when we bump kubebuilder to v5.x
+	// We will not adopt this change since it did not work and was a bug introduced in the
+	// default Makefile of kubebuilder v4.x.
+	makefileTestE2ETarget = `@command -v $(KIND) >/dev/null 2>&1 || { \
+		echo "Kind is not installed. Please install Kind manually."; \
+		exit 1; \
+	}
+	@$(KIND) get clusters | grep -q 'kind' || { \
+		echo "No Kind cluster is running. Please start a Kind cluster before running the e2e tests."; \
+		exit 1; \
+	}
```

Comment:
@acornett21 just for you be aware of.
It should not be added to the scaffold.
We changed already on master, so it is fixed. I am cleaning up.